less/hotgraphic.less

```diff
@@ -1,80 +1,111 @@
-.hotgraphic {
-  &__widget {
-    position: relative;
-  }
+.hotgraphic__widget {
+  position: relative;
+}
 
-  // Hotgraphic as pins
-  // --------------------------------------------------
-  &__pin {
-    position: absolute;
-    top: 0;
-    left: 0;
-    padding: @icon-padding / 2;
+// Hotgraphic as pins
+// --------------------------------------------------
+.hotgraphic__pin {
```

Comment:
@guywillis Good callout. I initially wanted to split things up into more distinct sections, but it definitely could use some tidying up.

I have just pushed a fix, so that should remove some of the redundancies in the styles.